app/actions/balancesActions.js

```diff
@@ -21,23 +21,42 @@ async function getBalances({ net, address, tokens }: Props) {
     endpoint = await api.getRPCEndpointFrom({ net }, api.neoscan)
   }
 
-  // token balances
-  const tokenBalances = await api.nep5.getTokenBalances(
-    endpoint,
-    tokens
-      .filter(token => !token.isUserGenerated)
-      .map(token => token.scriptHash),
-    address
+  const chunks = tokens
+    .filter(token => !token.isUserGenerated)
+    .reduce((accum, currVal, index) => {
+      if (!accum.length) {
+        accum.push([currVal.scriptHash])
+        return accum
+      }
+
+      if (accum[accum.length - 1].length < 4) {
```

Comment:
why 4? a good variable name would help

Reply:
Thanks for taking a look guys... so 4 represents the "maximum chunk size" I will go ahead and make this a constant